meter_test.go

```diff
@@ -220,11 +221,21 @@ func TestMetricsGetJSON(t *testing.T) {
 	counter2.Set(42)
 	require.Nil(t, metrics.Register("counter2", counter2))
 
-	b, err := metrics.GetJSON("count*")
+	b, err := metrics.GetJSON(func(key string) bool {
```

Comment:
I'd suggest to add a test where zero counters are suitable for our pattern.

Reply:
I've added tests for an empty pattern.